libs/pages/elewa/about-us/src/lib/components/mission/mission.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="mission">
+  <h4>We care!</h4>
+  <div class="mission-statements">
+    <p>
+      Elewa is a mission-driven organization. We make use of our cooperative and shared culture to drive the needle for the development of people and our environment. We care for our own, but also care deeply about the context surrounding us.
+    </p>
+    <p>
+      Our investments are therefore not limited to internal ones but contributes heavily to our community and environment. From training the next scout leaders on sustainable practices, to bridging the employment gap for junior software developers.
+    </p>
+  </div>
+  <div>
+    <elewa-group-elewa-vertical-icon-and-text></elewa-group-elewa-vertical-icon-and-text>
```

Comment:
This is not proper implementation of the reusable component, the loop should happen here.